### PR TITLE
[RDU-2019] add jonell taylor remove Logan

### PR DESCRIPTION
**Pull request by normalfaults**
[RDU-2019] add jonell taylor remove Logan

### CODE REVIEW

File: content/events/2019-raleigh/speakers/jonell-taylor.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
++++
+Title = "Jonell Taylor"
+Twitter = "JonellTaylor_"
+image = "jonell-taylor.jpg"
```
Comment (bridgetkromhout) [inline content/events/2019-raleigh/speakers/jonell-taylor.md]: This line shouldn't be populated unless you also add that file.

**Comment by bridgetkromhout** [comment]
Is Logan still speaking? If not, please also remove https://github.com/devopsdays/devopsdays-web/blob/master/content/events/2019-raleigh/speakers/logan-daigle.md and https://github.com/devopsdays/devopsdays-web/blob/master/content/events/2019-raleigh/program/logan-daigle.md and https://github.com/devopsdays/devopsdays-web/blob/master/static/events/2019-raleigh/speakers/logan-daigle.png.

File: content/events/2019-raleigh/speakers/jonell-taylor.md (hunk 1 above)
Comment (normalfaults) [reply content/events/2019-raleigh/speakers/jonell-taylor.md]: I added that file in a previous pull request: https://github.com/devopsdays/devopsdays-web/pull/8414

**Comment by normalfaults** [comment]
I will update the pull request with the logan files removed.

**Pull request merged**